app/controllers/notifications_controller.rb

```diff
@@ -26,33 +31,14 @@ def fetch_notifications
   end
 
   def signed_in
-    if !user_signed_in?
-      signed_in = -1
```

Comment:
So this code is being accessed in `notifications.js:197`. This was created to figure out whether the user is signed in from notifications.js.